realsense2_camera/src/rs_node_setup.cpp

```diff
@@ -22,6 +22,10 @@ using namespace rs2;
 
 void BaseRealSenseNode::setup()
 {
+#if defined (ACCELERATE_WITH_GPU)
+    bool use_gpu_processing = (_accelerate_with_gpu == accelerate_with_gpu::GL_GPU);
+    initOpenGLProcessing(use_gpu_processing);
```

Comment:
can we save this call if `accelerate_with_gpu != GL_GPU` ?
or, we must call it and init things there ?

Reply:
> or, we must call it and init things there ?

We must call `initOpenGLProcessing()` and init things even for `accelerate_with_gpu != GL_GPU` case, because the filter objects are created from `rs::gl::colorizer`, `rs2::gl::align` and `rs2::gl::pointcloud`. classes